Cooper/ClientApp/src.v2/app/components/foreign-profile/foreign-profile.component.html

```diff
@@ -0,0 +1,21 @@
+<div>
+  <mat-card class="avatar-card">                
+    <img mat-card-image src="{{profile.photoURL}}" onError="this.src='assets/imageKeeper/robot.png'" class="avatar-img">
+    <mat-card-title>{{profile.nickname}}</mat-card-title> 
+  </mat-card>
+
+  <mat-card-content class="info-card">
+    <h5>Name:</h5>
+      <p class="info-card__field" >{{profile.name}} </p>
```

Comment:
no BEM

Reply:
I like BEM because it has strict logic to follow)) 

And it doesn't get to mind how to name class correctly without using BEM. 

Css naming rules are a bit scaring to me :|